packages/dd-trace/src/opentelemetry/logs/otlp_http_log_exporter.js

```diff
@@ -11,7 +11,7 @@ const OtlpTransformer = require('./otlp_transformer')
 /**
  * OtlpHttpLogExporter exports log records via OTLP over HTTP.
  *
- * This implementation follows the OTLP HTTP specification:
+ * This implementation follows the OTLP HTTP v1.7.0 specification:
```

Comment:
Could you move the `@opentelemetry/resources` dependency to dev dependencies? It is only used as type for us. The same applies to `@opentelemetry/api-logs`.

Reply:
I opened this PR to move `opentelemetry/resources`: https://github.com/DataDog/dd-trace-js/pull/6883. 

`api-logs` is used here, we can't make it a dev dependency: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/opentelemetry/logs/logger_provider.js#L2

Reply:
I think we should use it as a peer dependency instead of a regular one.